website/docs/docs/deploy/dbt-cloud-job.md

```diff
@@ -37,6 +37,19 @@ Refer to [Scheduler](/docs/deploy/job-scheduler) for more info on how the schedu
 
 <Lightbox src="/img/docs/dbt-cloud/deployment/deploy-scheduler.jpg" width="100%" title="An overview of a dbt Cloud job run"/>
 
+### Deployment monitor
+
+Use the Deployment monitor to gain greater visibility into your deployment health and quickly detect any issues with the improved filtering and sorting capabilities. You can now easily curate the view based on environment, prioritizing the ones that matter most, such as Production. With pre-filtered lists for in progress and in queue activities, it's quicker and easier to track the relevant activity and stay on top of your deployment status.
+
+The Deployment monitor is accessible by going to the main dbt Cloud dashboard page or clicking the dbt logo on the upper-left. It includes:
+
+- A new run history visualization component to quickly scan a larger number of recent runs and get a better overview of overall health/activity 
+- Pre-filtered lists have been added for In Progress, In Queue, and Latest Failed jobs. 
```

Comment:
sugg: i thiiiink you're referring to the "cards" in the Deploy Monitor view/page so bolding since it's a UI label

```suggestion
- Prefiltered lists for **In Progress**, **In Queue**, and **Latest Failed** jobs. 
```